packages/viz/examples/styles/default_styles/lines.html

```diff
@@ -29,11 +29,11 @@
     <script src="https://unpkg.com/deck.gl@8.1.0/dist.min.js"></script>
 
     <!-- CARTO Deck.gl's Tile Layer -->
-    <script src="/dist/viz.umd.js"></script>
+    <script src="/dist/umd/index.min.js"></script>
 
     <script>
       async function initialize() {
-        carto.setDefaultCredentials({ username: 'public' });
+        cartoViz.setDefaultCredentials({ username: 'public' });
```

Comment:
we'll move this in the future to upper folder for examples, and then we'll use `carto.setDefaultCredentials` + `carto.viz.Layer` ...